product_shelf_info/report/report_deliveryslip.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="report_delivery_document" inherit_id="stock.report_delivery_document">
```

Comment:
I don't think this template is working. I tried printing a Delivery Slip from stock.picking and it did not show. (Product has `shelf_id`).